codemods/mocha/vitest/migrate-configuration/src/index.ts

```diff
@@ -149,6 +154,26 @@ export const repomod: Filemod<Record<string, never>, Record<string, never>> = {
 			};
 		}
 
+		if (path.endsWith('.gitignore')) {
+			const expressions = data.split('\n');
+
+			if (
+				expressions.some(
+					(expression) => expression.trim() === 'coverage',
+				)
```

Comment:
mb `includes('coverage')`?

Reply:
I think we should target the `coverage` itself, not `coverage/a` for instance.

Reply:
i mean cases like `**/coverage`, `project-name/coverage`

Reply:
so when it ends with coverage, it's correct?

Reply:
hm, yeah, probably endsWith should do